.github/dependabot.yml

```diff
@@ -9,3 +9,9 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "daily"
+
+  - package-ecosystem: "terraform"
+    directory: "terraform/groups/ecs-service"
+    schedule:
+      interval: "daily"
+    versioning-strategy: "increase"
```

Comment:
The `versioning-strategy` attribute isn't supported for the Terraform ecosystem so this can be removed, and a `daily` frequency is likely to generate excessive pull requests—`weekly` should suffice for these kind of updates.

You'll also need to introduce configuration for a registry to support Terraform updates for this project as it uses modules from a private [terraform-modules](https://github.com/companieshouse/terraform-modules) GitHub repository [here](https://github.com/companieshouse/limited-partnerships-web/blob/e3b4d75ca47b034ba37be526c4d0a3252d8b651e/terraform/groups/ecs-service/main.tf#L22) and [here](https://github.com/companieshouse/limited-partnerships-web/blob/e3b4d75ca47b034ba37be526c4d0a3252d8b651e/terraform/groups/ecs-service/main.tf#L31). 

I would also recommend grouping updates (per package manager) as it can reduce the number of open pull requests and maintenance burden, generally making things easier to manage:  

```suggestion
  - package-ecosystem: "terraform"
    directory: "terraform/groups/ecs-service"
    schedule:
      interval: "weekly"
registries:
  github-dependabot:
    type: git
    url: https://github.com
    username: x-access-token
    password: ${{secrets.DEPENDABOT_GITHUB_ACCESS_TOKEN}}
    groups:
      version-updates:
        applies-to: "version-updates"
        patterns:
          - "*"
      security-updates:
        applies-to: "security-updates"
        patterns:
          - "*"
```

This would require the introduction of a `DEPENDABOT_GITHUB_ACCESS_TOKEN` Dependabot secret, either at the repository or organisation level. We've previously implemented this across several projects at the repository level (prior to the migration to GitHub Enterprise) but this would be best implemented as an organisation secret now.

---

The main issue with adopting Dependabot is the lack of ownership across projects in general—I can see from the existing configuration for the `npm` ecosystem that updates [aren't being merged](https://github.com/companieshouse/limited-partnerships-web/pulls?q=is%3Apr+is%3Aclosed), instead Dependabot is closing them and replacing them as newer versions of dependencies become available. It will eventually disable itself after a 90 day window if it goes unmaintained. This is something you'll want to escalate as we've already seen this in other projects, where Dependabot pull requests are ignored and have no real meaningful value as a result.